common/flatpak-context.c

```diff
@@ -1496,6 +1569,39 @@ option_remove_generic_policy_cb (const gchar *option_name,
   return TRUE;
 }
 
+static gboolean
+option_usb_cb (const gchar  *option_name,
+               const gchar  *value,
+               gpointer      data,
+               GError      **error)
+{
+  FlatpakContext *context = data;
+
+  if (!flatpak_context_parse_usb_device (context, value, error))
+    return FALSE;
+
+  flatpak_context_add_usb_device (context, value);
+  return TRUE;
+}
+
+static gboolean
+option_no_usb_cb (const gchar  *option_name,
+                  const gchar  *value,
+                  gpointer      data,
+                  GError      **error)
+{
+  g_autofree char *extended_value = NULL;
+  FlatpakContext *context = data;
+
+  if (!flatpak_context_parse_usb_device (context, value, error))
+    return FALSE;
+
+  extended_value = g_strconcat ("!", value, NULL);
```

Comment:
Instead of keeping a single list, and using the `!` prefix, how about just keeping an `allowed_usb_devices` and `blocked_usb_devices` separately?

Reply:
I don't know. I just followed the pre-existing convention in flatpak-context.c, apparently this is the common way to handle it. Not sure if it would be better, but I can try and see how it looks.

Reply:
I saw policies do this which I might also not like.

Juggling the `"!"` around seems less clear and more tedious to me.

Reply:
It looks no better nor worse IMHO, but we're dealing with arbitraries, let's go with yours :slightly_smiling_face:

Reply:
Done. How do you feel about it now?

Reply:
Looks good.